profiler/proftest/proftest.go

```diff
@@ -472,6 +472,10 @@ func parseBackoffDuration(line string) (time.Duration, error) {
 func (tr *GCETestRunner) PollForAndReturnSerialOutput(ctx context.Context, inst *InstanceConfig, finishString, errorString string) (string, error) {
 	var output string
 	defer func() {
+		// Avoid escaping and double newlines in the rendered output (b/175999077).
+		// TODO: Use strings.ReplaceAll once no need to support Go 1.11.
```

Comment:
once support for Go 1.11 is dropped 
or
once there is no need to support Go 1.11

Reply:
Fixed.